src/data_tools/include/data_tools/jsf_data.h

```diff
@@ -43,13 +46,13 @@ struct jsf_sss_ping
     long long time_stamp_; // posix time stamp
     jsf_sss_ping_side port;
     jsf_sss_ping_side stbd;
-    bool is_bad;
+    int channel_num; // channel number, 0 -> port, 1 -> starboard
```

Comment:
Why do you have both `port` and `stbd` channels as members and then also a flag for checking which channel number?

Reply:
It could be a little redundant, but for me it is easier to check the channel number than checking members, to detect if there is a bad ping and to form the pings in the end. Or do you think we should create a function like "is_null" in jsf_sss_ping_side instead?